src/app/review/review.component.ts

```diff
@@ -70,13 +90,63 @@ export class ReviewComponent implements OnInit {
                 this.coursesReviewed = Promise.resolve(coursesReviewed);
             });
             this.questions = this.reviewService.getQuestions();
-        } else { // Other user types
+        } else if(this.userType == 2){
+            this.user = this.teacherService.getTeacher(this.userID);
+            Promise.all([
+                this.courses = this.teacherService.getCoursesByTeacher(this.userID),
+                this.reviews = this.teacherService.getReviewsByTeacher(this.userID),
+            ]).then(data => {
```

Comment:
This logic needs here to be slightly modified.  We don't need a list of courses that needs to be reviewed.  Instead we just need to display the reviews that have already taken place.

for example:

```
const coursesReviewed = [];
reviews.forEach(review => {
    coursesReviewed.push(review);
});
```

The structure of the return will probably need to be modified to account for the expected values in the html file.